vuepress/docs/next/tutorials/devops/installation/open-shift/openshift-install-by-operator-hub.md

```diff
@@ -262,6 +262,30 @@ Now we'll create the application itself.
 
 See the [Next Steps](#next-steps) below to continue your work with Entando.
 
+## Appendix A - Troubleshooting and Common Errors
+
+### SSL Certificate Error
+When installing Entando V6.3.2 into OpenShift 4.6, you may run into a SSL certificate error. This happens because of restricted registries for the docker image. 
+
+To address this issue, a property can be used to override the default docker registry, made available in a ConfigMap.  Then every time there is a docker image name without a registry, it will use this instead of the docker.io. 
```

Comment:
I'd switch up the placement of "can be": "To address this issue, a property used to override the default docker registry can be made available in a ConfigMap."
I'd also nix "then" and start the next sentence with "every".

Reply:
addressed awkward language.